packages/@aws-cdk/toolkit-lib/lib/actions/bootstrap/index.ts

```diff
@@ -169,6 +169,25 @@ export interface BootstrapParameters {
   readonly customPermissionsBoundary?: string;
 }
 
+export interface EnvironmentBootstrapResult {
+  environment: string;
+  status: 'success' | 'failed' | 'no-op';
```

Comment:
See other comment, `failed` will just never be returned.

```suggestion
  status: 'success' | 'no-op';
```